TestON/tests/FuncStartTemplate/FuncStartTemplate.py

```diff
@@ -0,0 +1,231 @@
+
+# Testing the basic functionality of ONOS Next
+# For sanity and driver functionality excercises only.
+
+import time
+import json
+
+time.sleep( 1 )
```

Comment:
must be a typo I have it in mine too.

On Tue, May 19, 2015 at 11:27 AM, Jon Hall <notifications@github.com> wrote:

> In TestON/tests/FuncStartTemplate/FuncStartTemplate.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/117#discussion_r30630442>
> :
>
> > @@ -0,0 +1,231 @@
> > +
> > +# Testing the basic functionality of ONOS Next
> > +# For sanity and driver functionality excercises only.
> > +
> > +import time
> > +import json
> > +
> > +time.sleep( 1 )
>
> why are you sleeping here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/117/files#r30630442>.
>